### ISSUE TITLE
Is it a joke? MECX futures login

### ISSUE THREAD

**Issue by KindSpidey** [ccxt/ccxt#13255]
I have code
```
mexc_futures = ccxt.mexc({
    "apiKey": config.MEXC_API_KEY,
    "secret": config.MEXC_SECRET_KEY,
    "options": {'defaultType': 'future' }
})
```
And got
```
ccxt.base.errors.ExchangeError: mexc does not support 'future' type, set exchange.options['defaultType'] to 'spot', 'margin', 'delivery' or 'future'
```
What am i supposed to do?)

**Comment by kroitor** [comment]
@KindSpidey sorry, it's a misleading erroneous message, it actually supports either `spot` or `swap` (for perpetual futures aka swaps). In this case, you should use `swap` instead of `future`. Will fix the message in the upcoming build. Thank you for pointing it out.

**Comment by KindSpidey** [comment]
Thanks, now it works

**Comment by kroitor** [comment]
@KindSpidey i've pushed a fix to that message in CCXT 1.82.53 (the new build will arrive in 20 minutes). Thank you for your feedback!
